Fix various typos (#2568)
nest/nest-simulator: doc/htmldoc/faqs/faqs.rst

   which compiler was used. Then re-build NEST with the same compiler
   version.

> 6. **I get a segmentation fault where I use SciPy in the same script
>    together with PyNEST**. We recently observed that if PyNEST is used

Reviewer: ```suggestion
6. **I get a segmentation fault when I use SciPy and PyNEST in the same script**. We recently observed that if PyNEST is used
```

I believe the *where* should be a *when*. I also suggest a more concise formulation.
